aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -373,6 +373,34 @@ Tensor& logit_(Tensor& self, c10::optional<double> eps) {
   return at::logit_out(self, self, eps);
 }
 
+Tensor& nan_to_num_out(
+    Tensor& result,
+    const Tensor& self,
+    c10::optional<double> nan,
```

Comment:
Should this be named 'num' (since it's "nan to num")?

Reply:
Ug... but you're correct because in NumPy it's named "nan." Not a great name but you're right we should be consistent.